process/dataValidators/nilHeaderValidator_test.go

```diff
@@ -24,14 +24,3 @@ func TestNilHeaderValidator_IsHeaderValidForProcessing(t *testing.T) {
 
 	assert.Nil(t, nhv.HeaderValidForProcessing(nil))
 }
-
-//------- IsInterfaceNil
-
-func TestNilHeaderValidator_IsInterfaceNil(t *testing.T) {
-	t.Parallel()
-
-	hdrValidator, _ := dataValidators.NewNilHeaderValidator()
```

Comment:
please replace with declaration and leave the assert

Reply:
here i cannot replace with a declaration because NewNilHeaderValidator will return a private type ... so to fix the warning i will write this line of code " _ = hdrValidator"